sdk/servicebus/service-bus/samples-dev/topicSubscriptionWithRuleOperationsSample.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates the topic filter functionality in Service Bus.
+ * It follows similar functionality to TopicSubscriptionWithRuleOperationsSample/program.cs
+ *
+ * https://github.com/Azure/azure-service-bus/blob/master/samples/DotNet/GettingStarted/Microsoft.Azure.ServiceBus/TopicSubscriptionWithRuleOperationsSample/Program.cs
+ *
+ * @summary Demonstrates how to filter messages in Service Bus
+ * @azsdk-weight 100
+ */
+
+import {
+  ServiceBusClient,
+  ServiceBusAdministrationClient,
+  ServiceBusMessage,
+} from "@azure/service-bus";
+import { DEFAULT_RULE_NAME } from "../src/util/constants";
```

Comment:
Let's defined the constant here instead of importing from source code.  We also publish these samples in standalone zip file at docs.microsoft.com.